libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -211,10 +211,7 @@ void AC_Avoid::adjust_velocity_z(float kP, float accel_cmss, float& climb_rate_c
 // roll and pitch value are in centi-degrees
 void AC_Avoid::adjust_roll_pitch(float &roll, float &pitch, float veh_angle_max)
 {
-    // exit immediately if proximity based avoidance is disabled
-    if ((_enabled & AC_AVOID_USE_PROXIMITY_SENSOR) == 0 || !_proximity_enabled) {
-        return;
-    }
+    bool adjusted = false;
```

Comment:
you could put that on line 226 to reduce its scope in case we return early on line 218 !

Reply:
OK, I got it. :)